doc/sphinx/language/core/records.rst

```diff
@@ -135,11 +135,13 @@ synonym :cmd:`Structure`.
 
       .. coqtop:: all
 
-         Record foo := { bar : nat } as HIMOM.
-         About bar.
-         Arguments bar {HIMOM}.   (* make "foo" an implicit parameter *)
-         Check bar.
-         Check (bar (HIMOM:={|bar:=0|})).
+         Record MyRecord := { myfield : nat } as VarName.
```

Comment:
Surely there is a less generic name than "VarName"?  How about "MyRecordType"?

Reply:
I chose the generic name to really hammer home what it actually was. I feel like calling it `MyRecordType` would muddy the waters.